lib/rules/strict-component-boundaries.js

```diff
@@ -32,6 +38,20 @@ module.exports = {
   },
 };
 
+function inNodeModules(importSource, context) {
+  const resolvedSource = resolve(importSource, context);
+
+  if (resolvedSource === null) {
+    return true;
+  }
+
+  if (!resolvedSource) {
```

Comment:
What do the two checks above do?

Reply:
`resolvedSource  === null` if it is a core library
`resolvedSource === undefined` if it is not installed